src/test/java/com/wikia/webdriver/testcases/adstests/TestFloorAdhesion.java

```diff
@@ -0,0 +1,90 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.dataprovider.mobile.MobileAdsDataProvider;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.common.templates.TemplateDontLogout;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.mobile.MobileAdsBaseObject;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+public class TestFloorAdhesion extends TemplateDontLogout {
+    private final String FLOOR_ADHESION_SELECTOR = "#ext-wikia-adEngine-template-footer";
+    private final String FLOOR_ADHESION_AD_SELECTOR = "#ext-wikia-adEngine-template-footer .ad";
+    private final String
+            FLOOR_ADHESION_CLOSE_SELECTOR = "#ext-wikia-adEngine-template-footer .close";
+    private final String WIKIA_BAR_SELECTOR = "#WikiaBar";
+
+    @Test(
+            dataProvider = "floorAdhesionOnOasis",
+            dataProviderClass = AdsDataProvider.class,
+            groups = {"TestFloorAdhesionOasis"}
+    )
+    public void testFloorAdhesionOnOasis(
```

Comment:
Yep, that's correct. The `TestFloorAdhesion` test isn't skin-dependent. With the right data provided it works for both Oasis and Mercury. Not using data provider here will result in having two separate tests for both skins with a lot of code duplication or having one test with a lot of if-skin logic which is :no_good: